tests/test_market.py

```diff
@@ -45,6 +45,14 @@
 }
 
 
+@pytest.fixture(scope="function", autouse=True)
+def device_registry_auto_fixture():
+    DeviceRegistry.REGISTRY = device_registry_dict
+    ConstSettings.IAASettings.MARKET_TYPE = 1
```

Comment:
If this test is dependent from the state of the system this should be set by default, is there something I'm missing ?

Reply:
Another test changes the state of the system to two sided market and does not revert it to the original value. I unfortunately had a hard time and did not manage to find out the culprit test, still though this should be here IMO. Since these tests require one-sided market in order to pass, it should be explicitly configured